docs/source/API/core/builtinreducers/ReducerConcept.rst

```diff
@@ -82,7 +85,12 @@ Functions
 
 .. cppkokkos:kokkosinlinefunction:: void init(value_type& val) const;
 
-    * Initialize ``val`` with appropriate initial value. For example, 'Add' assigns ``val = 0;``, but Prod assigns ``val = 1;``
+    * Optional callback initializing ``val`` with appropriate initial value. For example, 'Add' assigns ``val = 0;``, but Prod assigns ``val = 1;``.
+      Defaults to calling the default constructor.
+
+.. cppkokkos:kokkosinlinefunction:: void final(value_type& val) const;
+
+    * Optional callback modifying the result ``val``. Defaults to a no-op.
```

Comment:
Should we be explicit and say "no-op (that is do nothing)" or you think it is obvious enough and we use that terminology other places?

Reply:
We use it in a couple of other places so I think it's fine. I'm happy to change if requested, though.